Gruntfile.js

```diff
@@ -40,7 +40,6 @@ module.exports = function (grunt) {
 						util.format('mkdir -p %s', version_dir),
 						util.format('mkdir -p %s/logs', version_dir),
 						util.format('mkdir -p %s/pids', version_dir),
-						util.format('mkdir -p %s/public', version_dir),
```

Comment:
As discussed, we need this: `util.format('mkdir -p %s/public/dapps', version_dir),`